src/styles/components/container.scss

```diff
@@ -0,0 +1,46 @@
+.cards {
+  --columns-count: 4;
+  padding: 50px 40px 72px;
+  display: grid;
+  grid-template-columns: repeat(var(--columns-count), 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (max-width: 1023px) {
+  .cards {
+    --columns-count: 3;
+  }
+
+  .header {
+    padding-top: 10px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .nav__list {
+    display: flex;
+  }
+}
+
+@media (max-width: 768px) {
+  .cards {
+    --columns-count: 2;
+  }
+
+  .header {
+    padding: 10px 0;
+  }
+  .nav__list {
+    display: none;
+  }
+}
```

Comment:
Check these styles, they should not be in `container.scss` file, they should be moved in the files with the blocks styles